app/assets/stylesheets/main.css.scss

```diff
@@ -0,0 +1,8 @@
+.user-links,
+.navigation-links {
+  text-align: right;
+}
+
+#error_explanation {
+  font-weight: bold;
+}
```

Comment:
Will implement as a flash. I like it better and it will be consistent with the rest of the application, as you mentioned.